Gemfile

```diff
@@ -77,4 +75,6 @@ group :test do
   gem 'vcr'
   gem 'webmock'
   gem 'shoulda-matchers'
+  gem 'database_cleaner'
```

Comment:
That's me forgetting to re-add the rails_helper bits :/. But we defo use not transactional test when using the feature tests. Although there are "SystemTest" that removes the need for database cleaner it does not seem available to RSpec unfortunately at the moment.